src/datomic_rest_api/rest/object.clj

```diff
@@ -245,12 +248,11 @@
     (for [paper (:evidence/paper-evidence holder)]
       (pack-obj "paper" paper)))
 
-   :Date_last_updated
+   :Date_last_updated 
    (if-let [date (:evidence/date-last-updated holder)]     
-       { :id (str (.format (java.text.SimpleDateFormat. "yyyy-M-d") date))
-         :label (str (.format (java.text.SimpleDateFormat. "yyyy-M-d") date))
-         :class "text"})
-       
+       [{ :id (str (.format (java.text.SimpleDateFormat. "yyyy-M-dd") date))
+         :label (str (.format (java.text.SimpleDateFormat. "yyyy-M-dd") date))
+         :class "text"}])
```

Comment:
This is fine, but could probably use more clojure-like idioms by using the [clj-time.format](https://github.com/clj-time/clj-time) library, rather than using java.text.